base/uk.ac.stfc.isis.ibex.dae/src/uk/ac/stfc/isis/ibex/dae/DaeObservables.java

```diff
@@ -423,7 +422,7 @@ public DaeObservables() {
         simulationMode = obsFactory.getSwitchableObservable(new BooleanChannel(),
                 InstrumentUtils.addPrefix(DAE.endWith("SIM_MODE")));
         eventModeBufUsed = obsFactory.getSwitchableObservable(new DoubleChannel(),
-                InstrumentUtils.addPrefix(DAE.endWith("EVENTMODEBUFUSED")));
+        		InstrumentUtils.addPrefix(DAE.endWith("EVENTMODEBUFUSED")));
```

Comment:
Not sure why it looks like this here, in eclipse they seemed to all be aligned.

Reply:
That's why we don't use tabs. Different software shows them differently. Spaces are always consistent.

Reply:
I set Eclipse to insert 4 tabs instead of spaces (I was trying it out there to see if it did change the tabs to spaces) should I go back and change it again?

Reply:
I had the same problem a few minutes ago. I realised it wasn't actually putting spaces in.
In Eclipse you can go to:
- Window -> Preferences -> Java -> Code Style -> Formatter -> Edit...
- Then under General settings change the tab policy to Spaces only.
- Change the profile name.
- Apply and OK.
- Select all your text and press Ctrl+Shift+F (formats text according to the rules we just changed).

Yes, I think it's probably best to change them.

Reply:
Ok great, thanks a lot!